HelenEsterman/kneel-server
# added get functionality for metals

I created a views directory in which I put an `MetalView` class. It handles all interactions with the database to get metal-related data.

Supported routes

`/metals` Will return all metal information
`/metals/n` Will return all metal information for the metal that has the same `id` as the *n*  in `metals/n`
##Steps to test
###All Metals
1. Pull down the `he-metal-getMethod` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Request `http://localhost:8000/metals` and verify that you get an array of metals in the response, and that each one has the following keys on it
    -id
    -metal
    -price
5. Also verify that the response code is 200
###Single Metal
1. Request `http://localhost:8000/metals/2` and verify that you get a metal object in the response, and that it has the following keys on it
    -id
    -metal
    -price
2. Also verify that the response code is 200
